java/blend-renderer/README.md

```diff
@@ -35,14 +38,15 @@ To apply a `BlendRenderer` to a `RasterLayer`:
 6. Create the ArcGIS/samples directory, `mkdir ArcGIS/samples/raster`.
 7. You should now have the following directory on your target device, `/sdcard/ArcGIS/samples/raster`. We will copy the contents of the downloaded data into this directory. Note:  Directory may be slightly different on your device.
```

Comment:
```suggestion
7. You should now have the following directory on your target device, `/Android/data/com.esri.arcgisruntime.sample.blendrenderer/files/`. We will copy the contents of the downloaded data into this directory. Note:  Directory may be slightly different on your device.
```

Reply:
not committing because it needs more changes above

Reply:
will be handled in a follow up PR